scss/plugins/angular-ui-bootstrap-reskin/_theme.defaults.scss

```diff
@@ -63,8 +63,8 @@ $modal: (
 	maxWidth: 					90%,
 
 	sizes: (
-		sm: 					15em,
-		md:						30em,
+		sm: 					35em,
```

Comment:
i thought [nguibs modal](http://angular-ui.github.io/bootstrap/#/modal) would support all breakpoints from the bootstrap library (e.g. `xs`, `sm`, `md` and `lg`) and was about to flame you for not supporting size `xs`. but then i checked and neither do they... kinda disappointed i cant flame you

either way i checked something else and what you put in the `size` attribute 

```js
var modalInstance = $uibModal.open({
  templateUrl: 'myModalContent.html',
  controller: 'ModalInstanceCtrl',
  size: 'my-custom-size', // here
  }
});
```

is passed to the div with the `.modal-dialog` class

```html
<div class="modal-dialog modal-my-custom-size"> <!-- here -->
  <div class="modal-content" uib-modal-transclude="">...</div>
</div>
```

so it would be a good idea to also support *OUR* breakpoints adjectives, and by that i mean `mobile`, `tablet`, `desktop` and `large`

what do you think